modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -17,6 +17,7 @@ The following table describes the CSI drivers that are installed with {product-t
 |CSI driver  |CSI volume snapshots  |CSI cloning  |CSI resize
 
 |AWS EBS (Tech Preview) | ✅ | - | ✅
+|Ember-CSI (Tech Preview)| ✅ | ✅ | ✅ 
```

Comment:
@lpettyjo Per the doc https://kubernetes-csi.github.io/docs/drivers.html, looks like Ember-CSI doesn't support cloning and resizing. Openshift storage QE team didn't test this feature.  @gcheresh@redhat.com Do you happen to know this?

Reply:
Thanks for the correction. I was not aware that it was not tested.

Reply:
I got this confirmation from OSP dev that all three were supported: https://issues.redhat.com/browse/RHOSPDOC-37?focusedCommentId=15838357&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15838357

From the OSP writer:

> For the table, OSP dev confirms, yes, all three are supported:
> 
> - CSI volume snapshots
> - CSI cloning
> - CSI resize

So to confirm, if OCP QE did not test but OSP did, does that mean that we should mark cloning and resizing as unsupported?

Reply:
@bobfuru I don't know. OCP storage QE didn't test the ember-csi operator and I don't know which team tested it. Checked in an OCP4.8 cluster, found it's a community operator.